queue/lib/txgh-queue/supervisor.rb

```diff
@@ -0,0 +1,72 @@
+require 'txgh'
+
+module TxghQueue
+  class UnexpectedResponse < StandardError; end
+
+  class Supervisor
+    ERROR_HANDLERS = [
+      ErrorHandlers::ServerResponse,
+      ErrorHandlers::Github,
+      ErrorHandlers::Transifex,
+      ErrorHandlers::TxghErrors,
+      ErrorHandlers::StandardErrors
```

Comment:
This might warrant a comment that `StandardErrors` should always come last, given how it is used in find_error_handler_class_for